test/abstract_syntax_tree.coffee

```diff
@@ -108,6 +108,11 @@ NUMBER = (value) -> {
   value
 }
 
+STRING = (value) -> {
+  type: 'StringLiteral'
+  value
+}
+
```

Comment:
What’s this?

Reply:
Just another "AST shorthand" for string literals similar to the existing `ID()` and `NUMBER()` shorthands above